docs/quarantine.md

```diff
@@ -102,3 +102,30 @@ After merging this PR the test will be out of quarantine.
 [2]: https://www.thoughtworks.com/en-us/insights/blog/no-more-flaky-tests-go-team
 [3]: https://docs.gitlab.com/ee/development/testing_guide/flaky_tests.html#quarantined-tests
 [on testgrid]: https://testgrid.k8s.io/kubevirt-periodics
+
+# Test Lane Quarantine
+
+There can be cases where required test lanes are flaking in a clustered manner with a large
+number of tests failing each time. This can cause major delays to merging important pull
+requests and overload CI. Individual test case quarantining does not make sense in this case.
+
+A required test lane should be made optional if the following criteria are met:
+* The required test lane has a failure rate higher than 25% in the last 24 hours
+* More than ten individual test cases are causing the required test lane to fail
+* The SIG responsible for the required test lane is unable to deliver a fix for the
+flake within 24 hours
+
+The percentage impact of a flaky test lane can be measured by searching for a relevant
+error on the [CI search page](https://search.ci.kubevirt.io/).
```

Comment:
Please excuse me, but I don't know what the relevant error may be. Can we add an example, e.g. how can I find the current failure rate of the windows lane?

Reply:
ah, sorry. I did not notice that this paragraph is about finding occurrence of a flaky *test*, not a flaky lane.

Would you add the windows example to the bulllet point in https://github.com/kubevirt/kubevirt/pull/12494/files#diff-4ff5f85030a93d12b9e7929720bd3d43b5f73cba313b3e98a56dbefe2d2abfabR113 ?

I'd like to add an example in order to make it easier for someone like myself to find the current failure rate of a lane that seems problematic.

BTW, does this statistic include cases where the lane failed justly, because it was executed on a bad PR?

Reply:
> ah, sorry. I did not notice that this paragraph is about finding occurrence of a flaky _test_, not a flaky lane.

This logic can be applied to a flaky lane as well. 
> 
> Would you add the windows example to the bulllet point in https://github.com/kubevirt/kubevirt/pull/12494/files#diff-4ff5f85030a93d12b9e7929720bd3d43b5f73cba313b3e98a56dbefe2d2abfabR113 ?
> 
> I'd like to add an example in order to make it easier for someone like myself to find the current failure rate of a lane that seems problematic.
> 
> BTW, does this statistic include cases where the lane failed justly, because it was executed on a bad PR?

This statistic includes all failures for the lane so it is not fair to use this one for this case - thats where the CI search comes in to provide an accurate measure of the impact percentage.

Reply:
To me this is a strong reason to use the your new metric. For example, we currently have kubevirt-e2e-kind-sriov failing 51 times out of 100 attempts over the recent week https://camo.githubusercontent.com/ddcf87891214d0984a90a503ac3e4f47f264c9fdc80d78f793332c40d72e9443/68747470733a2f2f6b756265766972742e696f2f63692d6865616c74682f6f75747075742f6b756265766972742f6b756265766972742f6661696c65646a6f62312e737667
I think that @xpivarc would agree that this is not acceptable. It is unfair for everyone and sig-network should get to the bottom of it ASAP.  I find it better to base our policy on clear and objective metric.

Reply:
I agree 51 out of 100 is crazy.

Reply:
+1 to define on what bases the decision to quarantine lane will take place.